### DISCUSSION TITLE
General change

### DISCUSSION THREAD

**Discussion by frack113** [SigmaHQ/sigma-specification#36] [General]
As there has been a lot of discussion in various places, the purpose of this discussion is to make a summary

- [x] Normalize file names without version
- [ ] Make a branch for V2
- [ ] Make a CI/CD to generate a web version
- [ ] optional find a logo or a mascot

**Comment by thomaspatzke** [comment]
A project is great to manage such todos, I've created one [here](https://github.com/orgs/SigmaHQ/projects/1/views/1) 😉

**Comment by frack113** [comment]
I forget to close it 2 years ago 😄
